docs/component-testing.md

```diff
@@ -0,0 +1,138 @@
+## Testplane Component Testing (experimental)
+
+Almost all modern web interfaces are written using frameworks (React, Vue, Svelte, ...) to simplify the creation, reuse and composition of web components. It is important to test such components in isolation from each other to be sure that each component is doing its job correctly. Just like we write unit tests separately from integration tests. Testplane already supports testing components using [Storybook](https://storybook.js.org/) (via [@testplane/storybook](https://github.com/gemini-testing/testplane-storybook) plugin), but this tool is not relevant for all projects. Therefore, we have developed another component testing option that does not require the use of Storybook.
+
+### Implementation options for component testing
+
+Component testing is a type of testing in which the logic of a web component is tested in isolation from the web page in which it is used. In order to perform such a test, you need to be able to render the component correctly. [JSDom](https://github.com/jsdom/jsdom) is often used for this task (also used inside Jest), which renders web components using the Node.js virtual renderer, i.e. without using a real browser. On the one hand, it works faster (the browser does not launch), and on the other hand, it is less stable, since the checks are not performed in a real browser. The second popular solution is to use a very fast dev server [Vite](https://vitejs.dev/), which supports many frameworks (React, Vue, Svelte, ...) and is responsible for rendering components in isolation.
```

Comment:
> "in which"

"where"

> (also used inside Jest)

"it is also used inside of Jest"

> using the Node.js virtual renderer, i.e. without using a real browser

I dont get what that "i.e." (in example) means here. I think, we should use "that is" here instead.

> the browser does not launch

What does the browser not launch? We should use passive voice here: "the browser is not being launched"

Reply:
> "where"

Why I should use `where` here? It seems that the meaning is the same

> I dont get what that "i.e." (in example) means here. I think, we should use "that is" here instead.

About i.e. - https://en.wiktionary.org/wiki/i.e.